docker/build

```diff
@@ -29,7 +29,7 @@ ENTRYPOINT [ "/bin/bash", "-l", "-c" ]
 EOF
 
 rm -rf naomi
-git clone https://github.com/mrc-ide/naomi
+git clone --single-branch --branch mrc-2406 https://github.com/mrc-ide/naomi
```

Comment:
I've approved but not merged the naomi PR, so this will need coordinating on merge